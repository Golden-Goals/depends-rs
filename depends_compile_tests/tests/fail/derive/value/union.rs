use depends::derives::Value;

#[derive(Value)]
#[depends(unhashable)]
union Foo {
    bar: usize,
}

fn main() {}
