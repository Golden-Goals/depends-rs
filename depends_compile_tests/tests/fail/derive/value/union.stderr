error: expected `struct`
 --> tests/fail/derive/value/union.rs:5:1
  |
5 | union Foo {
  | ^^^^^
